include/mruby/presym/enable.h

```diff
@@ -38,8 +38,4 @@ enum mruby_presym {
   static const mrb_sym name[] = {__VA_ARGS__};                                \
   static void init_##name(mrb_state *mrb) {}
 
-/* use MRB_SYM() for E_RUNTIME_ERROR etc. */
-# undef MRB_E_SYM
-# define MRB_E_SYM(sym) MRB_SYM(sym)
```

Comment:
Even when `mruby/presym/enable.h` is included **after** `mruby.h` above lines do the trick (replace `MRB_E_SYM` with `MRB_SYM` version).